dependencies/pip/dev_requirements.txt

```diff
@@ -32,15 +32,6 @@ azure-storage-blob==12.19.1
     # via django-storages
 backcall==0.2.0
     # via ipython
-backports-zoneinfo[tzdata]==0.2.1 ; python_version < "3.9"
```

Comment:
I don't think this is a problem, but I'd appreciate your eyes on it, @noliveleger. See my related message on commit e4bc393a70a1041f3fb9dc8b244a7f540d6d95b5

Reply:
it is fine since - as you observed - it is readded by the script at the end of the file.